warehouse/query-core/src/main/java/datawave/query/tracking/ActiveQueryLog.java

```diff
@@ -0,0 +1,203 @@
+        }
+    }
+    
+    private void checkSettings(AccumuloConfiguration conf, boolean useDefaults) {
+        
+        String maxIdleStr = conf.get(MAX_IDLE);
+        if (maxIdleStr != null) {
+            try {
+                setMaxIdle(Long.valueOf(maxIdleStr));
+            } catch (NumberFormatException e) {
+                LOG.error("Bad value: (" + maxIdleStr + ") in " + MAX_IDLE + " : " + e.getMessage());
+            }
+        } else if (useDefaults) {
+            setMaxIdle(this.maxIdle);
+        }
+        
+        String logPeriodStr = conf.get(LOG_PERIOD);
+        if (logPeriodStr != null) {
+            try {
+                setLogPeriod(Long.valueOf(logPeriodStr));
+            } catch (NumberFormatException e) {
+                LOG.error("Bad value: (" + logPeriodStr + ") in " + LOG_PERIOD + " : " + e.getMessage());
+            }
+        } else if (useDefaults) {
+            setLogPeriod(this.logPeriod);
+        }
+
+        String logMaxQueriesStr = conf.get(LOG_MAX_QUERIES);
+        if (logMaxQueriesStr != null) {
+            try {
+                setLogMaxQueries(Integer.valueOf(logMaxQueriesStr));
+            } catch (NumberFormatException e) {
+                LOG.error("Bad value: (" + logMaxQueriesStr + ") in " + LOG_MAX_QUERIES + " : " + e.getMessage());
+            }
+        }
+
+        String windowSizeStr = conf.get(WINDOW_SIZE);
+        if (windowSizeStr != null) {
+            try {
+                setWindowSize(Integer.valueOf(windowSizeStr));
+            } catch (NumberFormatException e) {
+                LOG.error("Bad value: (" + windowSizeStr + ") in " + WINDOW_SIZE + " : " + e.getMessage());
+            }
+        }
+    }
+    
+    private Cache<String,ActiveQuery> setupCache(long maxIdle) {
+        Caffeine<Object,Object> caffeine = Caffeine.newBuilder();
+        caffeine.expireAfterAccess(maxIdle, TimeUnit.MILLISECONDS);
+        Cache<String,ActiveQuery> cache = caffeine.build();
+        return cache;
+    }
+    
+    synchronized public void remove(String queryId, Range range) {
+        ActiveQuery activeQuery = get(queryId);
+        activeQuery.finishRange(range);
+        if (activeQuery.getNumActiveRanges() == 0) {
+            this.CACHE.invalidate(queryId);
+        }
+    }
+    
+    synchronized public ActiveQuery get(String queryId) {
+        return this.CACHE.get(queryId, s -> new ActiveQuery(queryId, this.windowSize));
+    }
+    
+    class ActiveQueryTimerTask extends TimerTask {
+        @Override
+        public void run() {
+            synchronized (this) {
+                List<ActiveQuery> activeQueryList = new ArrayList<>();
+                for (ActiveQuery q : CACHE.asMap().values()) {
+                    activeQueryList.add(q);
+                }
+                activeQueryList.sort(activeQueryList, new TotalTimeComparator());
+
+                for (ActiveQuery q : CACHE.asMap().values()) {
+                    if (q.isInCall()) {
```

Comment:
Add an option to see all calls....

Reply:
Took out the check is isInCall, so all active queries are eligible to be displayed.